src/app/routes/learnerRoutes.js

```diff
@@ -43,6 +44,15 @@ module.exports = function (app) {
     healthService.checkDependantServiceHealth(['LEARNER', 'CASSANDRA']),
     proxyManagedUserRequest()
   )
+
+  app.get('/learner/user/v1/exists/email/:emailId',
+    googleService.validateRecaptcha
+  );
+
+  app.get('/learner/user/v1/exists/phone/:phoneNumber',
```

Comment:
what is the further process of this post the validateRecaptcha?

Reply:
- Route wildcards will handle request forward